cardano-node/cardano-node.cabal

```diff
@@ -162,10 +162,12 @@ executable cardano-node
   main-is:              cardano-node.hs
   ghc-options:          -threaded
                         -rtsopts
-  if arch(arm)
-    ghc-options:        "-with-rtsopts=-T -I0 -N1 -A16m --disable-delayed-os-memory-return"
-  else
-    ghc-options:        "-with-rtsopts=-T -I0 -N2 -A16m --disable-delayed-os-memory-return"
+
+  if impl(ghc >= 8.10)
```

Comment:
But look at what that does. This mean we use no args at all for earlier ghc versions.

I think it might actually work to use multiple `-with-rtsopts` like:
```
ghc-options:          "-with-rtsopts=-T -I0 -N1 -A16m"
if arch(arm)
  ghc-options:        "-with-rtsopts=-N1"
else
  ghc-options:        "-with-rtsopts=-N2"
if impl(ghc >= 8.10)
  ghc-options:        "-with-rtsopts=--disable-delayed-os-memory-return"
```

Reply:
Updated.  Thanks!